cnv/cnv_users_guide/cnv-edit-vms.adoc

```diff
@@ -0,0 +1,15 @@
+[id="cnv-edit-vms"]
+= Editing virtual machines
+include::modules/cnv_document_attributes.adoc[]
+:context: cnv-edit-vms
+toc::[]
+
+Edit a virtual machine by doing one of the following:
```

Comment:
~doing~ completing one of the following tasks:

Reply:
Done.